.github/workflows/ci.yaml

```diff
@@ -86,7 +88,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: changes
     if: |
-      (needs.changes.outputs.go == 'true') || (needs.changes.outputs.charts == 'true') || ${{ inputs.run_e2e }}
+      (needs.changes.outputs.go == 'true') || (needs.changes.outputs.charts == 'true')
```

Comment:
The `inputs.run_e2e` is used to manually run the e2e test workflow. If we remove this condition, not all steps will be executed.

Reply:
it was triggering the test for anything. I would say that maybe for e2e just on those corner case, people can do a stupid change on .go files :)

Reply:
i use it for testing and would like to keep it, when CI is broken on branches its nice to be able to just run github action e2e.